scripts/validation.sh

```diff
@@ -0,0 +1,7 @@
+# A simple script to validate project settings.
+
+if which swiftlint >/dev/null; then
```

Comment:
As another improvement to be part of this change, this script could be called from the script phase of the Xcode project (without the --strict param, I believe).

Reply:
that's a good idea, I will update this. The only thing is instead of using `LOCROOT` I will use config file to exclude sgen.

Reply:
I made an attempt to reuse the script. However, the XCode instead of showing warnings in IDE now just shows in phase execution script. I guess it sets some environment variables that doesn't get set correctly with this approach. Any other ideas to reuse this?

Reply:
Let's follow up offline on this. I thought of passing the root folder as a parameter to the script (which would be resolved from the script build phase. I don't see why the IDE wouldn't show the warnings... but we can experiment with that.